aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -8,23 +8,22 @@ export const ThemeModeToggle = () => {
   const label = darkMode.value ? 'Light Mode' : 'Dark Mode';
   const icon = darkMode.value ? <IconSun /> : <IconMoon />;
   return (
-    <Box align="start">
-      <Button onClick={() => darkMode.toggle()}>
-        <Box
-          background="blue!"
-          direction="row"
-          gap="xsmall"
-          justify="center"
-          overflow="hidden"
-          pad={{ vertical: 'small', horizontal: 'medium' }}
-          responsive={false}
-          round="xlarge"
-          width="small"
-        >
-          {icon}
-          {label}
-        </Box>
-      </Button>
+    <Box
```

Comment:
Curious as to the rationale for eliminating the `Button`?

Reply:
Good question, the focus was off due to the Button, it didn't adjust to the content of internal Box, so I moved the onClick inside the Box so the interactive element will be the Box and the focus will be accurate to the Box content, and then the Button didn't have any props and was just a redundant element in the DOM - so I removed the Button. 
We can take another approach of replacing the old Box rendering-approach with a Button, but I continued in the direction that was existing and usage of a single component. 
I'm willing to explore the latter.

Reply:
I was intrigued by this question and refactored the Box to be a Button wrapped with a Box, let me know what you think.